docs/api/supported-platforms.md

```diff
@@ -87,43 +67,88 @@ pnpm install @finos/fdc3
 </TabItem>
 </Tabs>
 
-The npm package provides a wrapper around FDC3, allowing you to use it with ES6 import syntax:
+There are two standardized types of interface to a DA that a web application may use (which is appropriate depends on where the web application is run):
 
-```javascript
-import * as fdc3 from '@finos/fdc3';
+- **Desktop Agent Preload**: Used where the Desktop Agent is able to inject the the `DesktopAgent` API at `window.fdc3` allowing an app to access it directly, for example in an Electron app or where a browser Browser Extension is in use.
+- **Desktop Agent Proxy**: Used when running in a standard web browser (without a browser extension or similar customization). The Desktop Agent will often be running in a different window or frame to the application and must be communicated with via cross-document messaging with `postMessage` and `MessagePorts` (see the [HTML5 Living Standard](https://html.spec.whatwg.org/multipage/web-messaging.html) for more details). A 'proxy' class implementing the Desktop Agent API is used to abstract the details of cross-document messaging, allowing the application to work with the FDC3 API directly.
```

Comment:
"must" should probably be "MUST"

Reply:
fixed in 976af80d5ba8d831e202aabcd37d1a7886e4c930